webcompat/form.py

```diff
@@ -80,44 +80,44 @@
 ]
 
 problem_choices = [
-    ('detection_bug', '<div class="icon-container"><img src="../../img/svg/icons/svg-problem-mobile-vs-desktop.svg" /></div> Desktop site instead of mobile site'),
-    ('site_bug', '<div class="icon-container"><img src="../../img/svg/icons/svg-problem-no-use.svg" /></div> Site is not usable'),
-    ('layout_bug', '<div class="icon-container"><img src="../../img/svg/icons/svg-problem-broken-design.svg" /></div> Design is broken'),
-    ('video_bug', '<div class="icon-container"><img src="../../img/svg/icons/svg-problem-no-video.svg" /></div> Video or audio doesn\'t play'),
-    ('unknown_bug', '<div class="icon-container"><img src="../../img/svg/icons/svg-problem-question.svg" /></div> Something else')
+    ('detection_bug', '<div class="icon-container"><img src="../../img/svg/icons/svg-problem-mobile-vs-desktop.svg" /></div> Desktop site instead of mobile site'),  # noqa
```

Comment:
Why  HTML is being used in there?

Reply:
[WTForms](https://wtforms.readthedocs.io/en/stable/fields.html) was used to generate the fields. For every option (in our case - the radio button), there is the possibility to specify only one label and a key. The options are defined in python. I am referring to these options on the [old site](https://www.screencast.com/t/DqOTQl9t) and the [new site](https://www.screencast.com/t/mBcEyDKQr97X). On the old site, there is only a text value and a key that is not visible. On the new site, each option has a text value, a key, and an image. Ideally, each option that comes from python should have these values. WTForms only supports two values (label and key), and that's why I created the HTML inside the Label. In this way, I am not sending only a text, but an image and a text altogether, in HTML format.

Reply:
You might want to use [Custom widgets](https://wtforms.readthedocs.io/en/stable/widgets.html#custom-widgets) then

Reply:
@karlcow I see, makes sense. @johngian I may need your help with this update.